src/models/events.js

```diff
@@ -17,13 +17,17 @@ import {
 const events = function events() {
   function createProjectProcedure() {
     const nameProject = document.getElementById('nameProject').value;
-    const ourStore = storage();
-    const newProject = createProject(nameProject);
-    ourStore.addProject(newProject);
-    const closeModalButton = document.getElementById('closeModalButton1');
-    closeModalButton.click();
-    document.getElementById('projectForm').reset();
-    dom().showProject();
+    if (nameProject !== '') {
+      const ourStore = storage();
+      const newProject = createProject(nameProject);
+      ourStore.addProject(newProject);
+      const closeModalButton = document.getElementById('closeModalButton1');
+      closeModalButton.click();
+      document.getElementById('projectForm').reset();
+      dom().showProject();
+    } else {
+      window.alert('You need to fill the input field!');
```

Comment:
I trust you can do this 👇  before merging, `required` is already doing the job
`This should be production code, we shouldn't have `alert`, please remove the `alert`.`